.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
we can also change two code-quality settings now:

* for pyupgrade to --py37-plus in the pre-commit yaml
* for black the target version to `['py37']` in pyproject.toml